src/bandersnatch/storage.py

```diff
@@ -340,7 +340,7 @@ def load_storage_plugins(
         config = BandersnatchConfig().config
     if not enabled_plugin:
         try:
-            enabled_plugin = config["mirror"]["storage-backend"]
+            enabled_plugin = config.get("mirror", "storage-backend")
```

Comment:
Why is this needed? Now we'll never hit to KeyError we catch to default to filesystem ...

Reply:
This is mainly for the test, the mocked config in tests only support `.get` method

Reply:
That is not a desirable thing to do when you break the actual codes logic just to make a mock to work.

What ever test is mocking here, can we just change it to read in once of the test config files into a ConfigParser object and change this back please.

Reply:
Actually you could just get rid of the `try/except` and make the .get default to "filesystem" right to get same behavior ...